src/components/Home/Header/index.tsx

```diff
@@ -114,94 +124,130 @@ const logosData: Array<{
     widthMd: 88,
     widthLg: 124,
     src: rabbitmqLogo,
-    alt: 'RabbitMQ logo'
+    alt: 'RabbitMQ logo',
+    link: '/doc/user-guide/serving/rabbitmq'
   },
   {
     widthSm: 66,
     widthMd: 88,
     widthLg: 124,
     src: pythonLogo,
-    alt: 'Python logo'
+    alt: 'Python logo',
+    link: '/doc/user-guide/building/pip'
   },
   {
     widthSm: 66,
     widthMd: 78,
     widthLg: 110,
     src: condaLogo,
-    alt: 'Conda logo'
+    alt: 'Conda logo',
+    link: '/doc/user-guide/building/conda'
   },
   {
     widthSm: 66,
     widthMd: 88,
     widthLg: 124,
     src: onnxLogo,
-    alt: 'ONNX logo'
+    alt: 'ONNX logo',
+    link: '/doc/user-guide/models'
```

Comment:
I think we need to add mentions of all ML frameworks to https://mlem.ai/doc/user-guide/models
So these links are meaningful.

Reply:
Agree, the existing model object-references pages e.g. https://mlem.ai/doc/object-reference/model/onnx are pretty useless as far as user docs go and not worth linking to from the front page